tests/system/action/committee/test_create.py

```diff
@@ -1 +1,120 @@
-# TODO
+from tests.system.action.base import BaseActionTestCase
+
+
+class CommitteeCreateActionTest(BaseActionTestCase):
+    def test_create(self) -> None:
```

Comment:
Ah, you should probably add one more test where only the required fields are given to make sure this works as intended.